.gitignore

```diff
@@ -2,5 +2,4 @@ logs/*
 .idea/
 node_modules
 npm-debug.log
-package-lock.json
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
The change here is to remove `package-lock.json` from the git ignore. Not sure why it wouldn't have been included.